doc/manual/manual.tex

```diff
@@ -143,7 +144,7 @@
   \begin{textblock*}{0in}(0.0in,0.8in)
     \begin{center}
       \vspace{1em}
-      \includegraphics[width=4.5in]{../logo/unlabeled_logo.pdf}
+      \includesvg[width=4.5in]{../logo/unlabeled_logo.svg}
```

Comment:
Did you forget to add this file to the PR?

Reply:
Oh, I had forgotten that I use that here. It's part of #4781. I can rebase this one on top of #4781 once that is merged.